install/check-minimum-requirements.sh

```diff
@@ -19,7 +20,13 @@ if [[ "$(vergte ${DOCKER_VERSION//v} $MIN_DOCKER_VERSION)" ]]; then
 fi
 echo "Found Docker version $DOCKER_VERSION"
 
-COMPOSE_VERSION=$($dc_base version --short)
+if [[ $($dc_base version --short) ]]; then
+  COMPOSE_VERSION=$($dc_base version --short)
+else
```

Comment:
```suggestion
COMPOSE_VERSION=$($dc_base version --short || echo '')
if [[ -z "$COMPOSE_VERSION" ]]; then
```
Same as above, I think this should be changed to  be consistent if you change above to follow BYK's suggestion.